src/calc-length.js

```diff
@@ -90,6 +90,70 @@
     return new CalcLength(calcDictionary);
   };
 
+  CalcLength.prototype._addCalcLengths = function(addedLength) {
+    if (!(addedLength instanceof CalcLength)) {
+      throw new TypeError('Argument must be a CalcLength');
+    }
+
+    var calcDictionary = {};
+
+    // Iterate through all possible length types and add their values
+    for (var i = 0; i < shared.LengthValue.LengthType.length; i++) {
+      var type = shared.LengthValue.LengthType[i];
+      if (this[type] == null) {
+        calcDictionary[type] = addedLength[type];
```

Comment:
If you do it this way around, addedLength[type] can still be null here. This sort of works, because
1 + null = null
null + 2 = 2

however
null + null = 0

So you will potentially end up with extra zeros in your output calclength.